{"uid":"50509619bbb0baa2","name":"Login with valid credentials test","fullName":"tests.test_login.TestLogin#test_valid_login","historyId":"e7f254b9bfd5f2bc8d4a1ffa6dc7aebb","time":{"start":1708131575548,"stop":1708131585979,"duration":10431},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55797d43cd93 <unknown>\n#1 0x55797d120337 <unknown>\n#2 0x55797d16bd60 <unknown>\n#3 0x55797d16be51 <unknown>\n#4 0x55797d1afbc4 <unknown>\n#5 0x55797d18e46d <unknown>\n#6 0x55797d1ad12d <unknown>\n#7 0x55797d18e1e3 <unknown>\n#8 0x55797d15e135 <unknown>\n#9 0x55797d15f13e <unknown>\n#10 0x55797d400e4b <unknown>\n#11 0x55797d404dfa <unknown>\n#12 0x55797d3ed6d5 <unknown>\n#13 0x55797d405a6f <unknown>\n#14 0x55797d3d169f <unknown>\n#15 0x55797d42a098 <unknown>\n#16 0x55797d42a262 <unknown>\n#17 0x55797d43bf34 <unknown>\n#18 0x7fa7c8a94ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f5c2eba5c10>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9c989bd3d23eae9eea76c6c186a5372e\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f5c2eb20900>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55797d43cd93 <unknown>\nE       #1 0x55797d120337 <unknown>\nE       #2 0x55797d16bd60 <unknown>\nE       #3 0x55797d16be51 <unknown>\nE       #4 0x55797d1afbc4 <unknown>\nE       #5 0x55797d18e46d <unknown>\nE       #6 0x55797d1ad12d <unknown>\nE       #7 0x55797d18e1e3 <unknown>\nE       #8 0x55797d15e135 <unknown>\nE       #9 0x55797d15f13e <unknown>\nE       #10 0x55797d400e4b <unknown>\nE       #11 0x55797d404dfa <unknown>\nE       #12 0x55797d3ed6d5 <unknown>\nE       #13 0x55797d405a6f <unknown>\nE       #14 0x55797d3d169f <unknown>\nE       #15 0x55797d42a098 <unknown>\nE       #16 0x55797d42a262 <unknown>\nE       #17 0x55797d43bf34 <unknown>\nE       #18 0x7fa7c8a94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1708131575546,"stop":1708131575546,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"secret_data","time":{"start":1708131575547,"stop":1708131575547,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1708131575547,"stop":1708131575547,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1708131575547,"stop":1708131575547,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55797d43cd93 <unknown>\n#1 0x55797d120337 <unknown>\n#2 0x55797d16bd60 <unknown>\n#3 0x55797d16be51 <unknown>\n#4 0x55797d1afbc4 <unknown>\n#5 0x55797d18e46d <unknown>\n#6 0x55797d1ad12d <unknown>\n#7 0x55797d18e1e3 <unknown>\n#8 0x55797d15e135 <unknown>\n#9 0x55797d15f13e <unknown>\n#10 0x55797d400e4b <unknown>\n#11 0x55797d404dfa <unknown>\n#12 0x55797d3ed6d5 <unknown>\n#13 0x55797d405a6f <unknown>\n#14 0x55797d3d169f <unknown>\n#15 0x55797d42a098 <unknown>\n#16 0x55797d42a262 <unknown>\n#17 0x55797d43bf34 <unknown>\n#18 0x7fa7c8a94ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f5c2eba5c10>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9c989bd3d23eae9eea76c6c186a5372e\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f5c2eb20900>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55797d43cd93 <unknown>\nE       #1 0x55797d120337 <unknown>\nE       #2 0x55797d16bd60 <unknown>\nE       #3 0x55797d16be51 <unknown>\nE       #4 0x55797d1afbc4 <unknown>\nE       #5 0x55797d18e46d <unknown>\nE       #6 0x55797d1ad12d <unknown>\nE       #7 0x55797d18e1e3 <unknown>\nE       #8 0x55797d15e135 <unknown>\nE       #9 0x55797d15f13e <unknown>\nE       #10 0x55797d400e4b <unknown>\nE       #11 0x55797d404dfa <unknown>\nE       #12 0x55797d3ed6d5 <unknown>\nE       #13 0x55797d405a6f <unknown>\nE       #14 0x55797d3d169f <unknown>\nE       #15 0x55797d42a098 <unknown>\nE       #16 0x55797d42a262 <unknown>\nE       #17 0x55797d43bf34 <unknown>\nE       #18 0x7fa7c8a94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1708131575548,"stop":1708131585978,"duration":10430},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55797d43cd93 <unknown>\n#1 0x55797d120337 <unknown>\n#2 0x55797d16bd60 <unknown>\n#3 0x55797d16be51 <unknown>\n#4 0x55797d1afbc4 <unknown>\n#5 0x55797d18e46d <unknown>\n#6 0x55797d1ad12d <unknown>\n#7 0x55797d18e1e3 <unknown>\n#8 0x55797d15e135 <unknown>\n#9 0x55797d15f13e <unknown>\n#10 0x55797d400e4b <unknown>\n#11 0x55797d404dfa <unknown>\n#12 0x55797d3ed6d5 <unknown>\n#13 0x55797d405a6f <unknown>\n#14 0x55797d3d169f <unknown>\n#15 0x55797d42a098 <unknown>\n#16 0x55797d42a262 <unknown>\n#17 0x55797d43bf34 <unknown>\n#18 0x7fa7c8a94ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"21de2b92d889ebd2","name":"public ip address","source":"21de2b92d889ebd2.txt","type":"text/plain","size":11},{"uid":"6130735dfdfffc30","name":"Cookies","source":"6130735dfdfffc30.json","type":"application/json","size":2},{"uid":"fdf04b030c0fa27b","name":"Session Storage","source":"fdf04b030c0fa27b.json","type":"application/json","size":2},{"uid":"979ba0586e291595","name":"Local Storage","source":"979ba0586e291595.json","type":"application/json","size":77},{"uid":"c16aaa4105d5e77f","name":"Console Logs","source":"c16aaa4105d5e77f.json","type":"application/json","size":599},{"uid":"6c52d35ebe8aa62f","name":"Full Page Screenshot","source":"6c52d35ebe8aa62f.png","type":"image/png","size":2477011},{"uid":"2a6be06d45bbce1d","name":"URL","source":"2a6be06d45bbce1d.uri","type":"text/uri-list","size":25},{"uid":"44dcefc61b6b06d8","name":"Network Logs","source":"44dcefc61b6b06d8.json","type":"application/json","size":1283739}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"tag","value":"Tagged test"},{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1538-984"},{"name":"thread","value":"2189-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","run(order=1)"]},"source":"50509619bbb0baa2.json","parameterValues":[]}